main.js

```diff
@@ -0,0 +1,33 @@
+const menuIconMobile = document.querySelector('.navbar__hamb');
+const menu = document.querySelector('.navbar__menu');
+const viewport = document.querySelector('body');
+const menuLinks = document.querySelectorAll('.navbar__menu__link');
+
+function isActiveClassExisted(element) {
+  return element.classList.contains('active');
+}
+
+function refreshIcon() {
+  if (isActiveClassExisted(menu)) {
+    menuIconMobile.setAttribute('src', './images/hamburguer-close.png');
+  } else {
+    menuIconMobile.setAttribute('src', './images/hamburguer.png');
+  }
+}
+
+function openMenu() {
+  menu.classList.toggle('active');
+  viewport.classList.add('blockover');
+  refreshIcon();
+}
+
+function closeMenu() {
+  menu.ClassList.remove('active');
+  viewport.classList.remove('blockover');
+  refreshIcon();
+}
```

Comment:
- [ ] When clicking on links it navigates to the right section 👏🏻, but let's kindly make sure it also closes the menu and turns scroll on again, basically should also trigger closeMenu function I believe 😇

Reply:
Thanks a lot for the encouraging feedback!  Working on the suggested change.

Reply:
Have had a sleepless night fixing that. Will keep improving and updating in the next windows popup version pls. Thanks.